fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -345,10 +341,62 @@ impl A380AirConditioningSystem {
                     ],
                 ),
             ],
-            cabin_fans: [CabinFan::new(ElectricalBusType::AlternatingCurrent(1)); 2],
+            tadd: TrimAirDriveDevice::new(
+                context,
+                vec![
+                    ElectricalBusType::AlternatingCurrent(2), // 117XP
+                    ElectricalBusType::AlternatingCurrent(4), // 206XP
+                ],
+            ),
+            vcm: [
+                VentilationControlModule::new(
+                    context,
+                    VcmId::Fwd,
+                    vec![
+                        ElectricalBusType::DirectCurrent(1),       // 411PP
+                        ElectricalBusType::DirectCurrentEssential, // 109PP
+                    ],
+                ),
+                VentilationControlModule::new(
+                    context,
+                    VcmId::Aft,
+                    vec![
+                        ElectricalBusType::DirectCurrent(2),       // 214PP
+                        ElectricalBusType::DirectCurrentEssential, // 109PP
+                    ],
+                ),
+            ],
+
+            cabin_fans: [
+                CabinFan::new(
+                    VolumeRate::new::<liter_per_second>(Self::CAB_FAN_DESIGN_FLOW_RATE_L_S),
+                    ElectricalBusType::AlternatingCurrent(1),
+                ), // Left Hand - 100XP1
+                CabinFan::new(
+                    VolumeRate::new::<liter_per_second>(Self::CAB_FAN_DESIGN_FLOW_RATE_L_S),
+                    ElectricalBusType::AlternatingCurrent(2),
+                ), // Left Hand - 100XP2
+                CabinFan::new(
+                    VolumeRate::new::<liter_per_second>(Self::CAB_FAN_DESIGN_FLOW_RATE_L_S),
+                    ElectricalBusType::AlternatingCurrent(3),
+                ), // Right Hand - 200XP3
+                CabinFan::new(
+                    VolumeRate::new::<liter_per_second>(Self::CAB_FAN_DESIGN_FLOW_RATE_L_S),
+                    ElectricalBusType::AlternatingCurrent(4),
+                ), // Right Hand - 200XP4
+            ],
```

Comment:
A small suggestion to reduce code duplication:
```suggestion
            cabin_fans: [
                ElectricalBusType::AlternatingCurrent(1), // Left Hand - 100XP1
                ElectricalBusType::AlternatingCurrent(2), // Left Hand - 100XP2
                ElectricalBusType::AlternatingCurrent(3), // Right Hand - 200XP3
                ElectricalBusType::AlternatingCurrent(4), // Right Hand - 200XP4
            ]
            .map(|powered_by| {
                CabinFan::new(
                    VolumeRate::new::<liter_per_second>(Self::CAB_FAN_DESIGN_FLOW_RATE_L_S),
                    powered_by,
                )
            }),
```

Reply:
Thanks for the suggestion, I sort of applied it but in a slightly different way as the `CabinFan` constructor has changed since this comment.